File: wheere-renewal-front/wheere/lib/model/dto/member_dto.dart
class MemberDTO {
  String mId;
  String mName;
  String mSex;
  String mBirthDate;
  String mNum;
  String fcmToken;

  MemberDTO({
    required this.mId,
    required this.mName,
    required this.mSex,
    required this.mBirthDate,
    required this.mNum,
    required this.fcmToken,
  });

  factory MemberDTO.fromJson(Map<String, dynamic> json) {
    return MemberDTO(
      mId: json["mId"],
      mName: json["mName"],
      mSex: json["mSex"],
      mBirthDate: json["mBirthDate"],
      mNum: json["mNum"],
      fcmToken: json['fcmToken'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      "mId": mId,
      "mName": mName,
      "mSex": mSex,
      "mBirthDate": mBirthDate,
      "mNum": mNum,
      "fcmToken": fcmToken,
    };
  }
}

File: wheere-renewal-back/wheere/src/main/java/kr/ac/kumoh/sjjpl138/wheere/member/dto/MemberDto.java
package kr.ac.kumoh.sjjpl138.wheere.member.dto;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@AllArgsConstructor
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class MemberDto {

    private String mId;
    private String mName;
    private String mSex;
    private LocalDate mBirthDate;
    private String mNum;
}
